CONTRIBUTING.md

```diff
@@ -15,43 +15,46 @@ Please note we have a code of conduct, please follow it in all your interactions
 # Contents
 
 - [Instructions](#instructions)
-  * [Notes on Creating New Buttons](#notes-on-creating-new-buttons)
-  * [Button Ideas](#button-ideas)
-  * [Create Pull Request](#create-pull-request)
-    + [Note on abandoned pull requests](#note-on-abandoned-pull-requests)
-  * [Code of Conduct](#code-of-conduct)
-    + [Our Pledge](#our-pledge)
-    + [Our Standards](#our-standards)
-    + [Our Responsibilities](#our-responsibilities)
-    + [Scope](#scope)
-    + [Enforcement](#enforcement)
+  - [Notes on Creating New Buttons](#notes-on-creating-new-buttons)
+  - [Button Ideas](#button-ideas)
+  - [Create Pull Request](#create-pull-request)
+    - [Note on abandoned pull requests](#note-on-abandoned-pull-requests)
```

Comment:
Add below this a list item linking to the section you added

Reply:
Alright, I'd work on that.